patches/0002-Clarify-metainfo-license.patch

```diff
@@ -0,0 +1,25 @@
+From b2583585a67191604418929f310cb0b2df2fbade Mon Sep 17 00:00:00 2001
```

Comment:
I don't think this patch is necessary at all

Reply:
This patch is sent to upstream as an MR (along with the other patches) so I just want to simulate how it would behave after it gets merged.